cloud/monitoring.rst

```diff
@@ -80,6 +80,120 @@ Failovers happen to address hardware failure, as mentioned, and also for other r
 *  Situation
 *  Follower
 
+StatsD external reporting
+=========================
+
+Citus Cloud can send events to an external `StatsD <https://github.com/etsy/statsd>`_ server for detailed monitoring. Cloud sends the following statsd metrics:
```

Comment:
I would say “Citus Cloud” instead of just “Cloud” towards the end of that sentence